kitsune/search/v2/management/commands/es7_reindex.py

```diff
@@ -41,10 +55,13 @@ def handle(self, *args, **kwargs):
 
             percentage = kwargs["percentage"]
             total = count
-            if percentage < 100:
-                count = int(count * percentage / 100)
-                qs = qs[:count]
-            print("Indexing {}%, so {} documents out of {}".format(percentage, count, total))
+            if count := kwargs["count"]:
```

Comment:
Will `count` always be in kwargs?

Reply:
Yep, the `default=None` on L25 ensures it'll be there (but falsey).

Reply:
Oh but there is a bug here, even if `kwargs["count"]` is falsey, it'll be assigned to count, overwriting the previous value of with `None`